### PR TITLE
Add get customer

### PR DESCRIPTION
**Pull request by GulshanBaryah**

### CODE REVIEW

File: DFC.App.Account.Services.DSS/Services/DssService.cs [hunk 1]
```diff
@@ -45,14 +47,11 @@ public async Task<Customer> CreateCustomerData(Customer customerData)
...
                 return null;
 
-            var postData = new StringContent(
-                JsonConvert.SerializeObject(customerData),
-                Encoding.UTF8,
-                MediaTypeNames.Application.Json);
-            postData = AddDefaultRequestHeaders(postData);
-            postData.Headers.Add("version", _dssSettings.Value.CustomerApiVersion);
+            
+            var request = await BuidRequestMessage();
```
Comment (LiamWright) [inline DFC.App.Account.Services.DSS/Services/DssService.cs:51]: Customer data is no longer assigned to request, and BuildRequestMessage has a typo

File: DFC.App.Account.Services.DSS/Services/DssService.cs [hunk 2]
```diff
@@ -61,11 +60,77 @@ public async Task<Customer> CreateCustomerData(Customer customerData)
...
+            
+            customer = await GetCustomerDetail(customerId, request);
+            customer.Addresses = await GetCustomerAddressDetails(customerId, request);
+            customer.Contact = await GetCustomerContactDetails(customerId, request);
+
+            return customer;
+
+        }
+
+        private async Task<HttpRequestMessage> BuidRequestMessage()
```
Comment (LiamWright) [inline DFC.App.Account.Services.DSS/Services/DssService.cs:80]: Does this need to be async if it's not doing any async operations?

File: DFC.App.Account.Services.DSS/Services/DssService.cs (hunk 1 above)
Comment (GulshanBaryah) [reply DFC.App.Account.Services.DSS/Services/DssService.cs:51]: Fixed

File: DFC.App.Account.Services.DSS/Services/DssService.cs (hunk 2 above)
Comment (GulshanBaryah) [reply DFC.App.Account.Services.DSS/Services/DssService.cs:80]: We await the call so we can run it async

Comment (LiamWright) [reply DFC.App.Account.Services.DSS/Services/DssService.cs]: It doesn't need to be async if it's not doing anything async operations, in fact Resharper is complaining about that fact

Comment (LiamWright) [reply DFC.App.Account.Services.DSS/Services/DssService.cs]: Happy to approve if there's a future reason it needs to be?

**Pull request merged**